packages/sources/react/src/components/overlays/VtmnToast/VtmnToast.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import '@vtmn/css-alert/dist/index-with-vars.css';
+import clsx from 'clsx';
+import { VtmnButton } from '../../actions/VtmnButton';
+
+export interface VtmnToastProps
+  extends React.ComponentPropsWithoutRef<'dialog'> {
+  /**
+   * The toast's content
+   */
+  content: string;
+
+  /**
+   * Whether there is an info icon
+   */
+  withIcon?: boolean;
+
+  /**
+   * Whether there is a close button
+   */
+  withCloseButton?: boolean;
```

Comment:
you don't need this for me. Why? Because you already have `onClose`. So if `onClose` is present, then you need to display a close button. If not, you don't need it, no?